phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/library/LibraryController.java

```diff
@@ -167,4 +191,30 @@ public void updateLibrary() {
     public LibraryView getView() {
         return libraryView;
     }
+
+    private void setDefaultCategoryIcons(List<CategoryDTO> categoryDTOS) {
```

Comment:
This method looks the same as the one in `AppsController` to me.
Maybe we should move it to a common class, so that we implement it only once?
A good location for the method would be the `ThemeManager` class in my opinion.

Reply:
Yes, I know. I wanted to handle this later because I'm not quite sure yet how to do it. For example, it might make sense to store the category icons which are required for the shortcuts separately such that you do not need the repository.

Reply:
What do you mean with repository?
Do you mean that you don't need an online connection to create the css or something else?

Reply:
I'll open an issue. Let's discuss it there.

Reply:
Ok